server/app.js

```diff
@@ -1,13 +1,66 @@
 const express = require('express');
 const cors = require('cors');
+const passport = require('passport');
+const GoogleStrategy = require('passport-google-oauth').OAuth2Strategy;
 const db = require('./db-connection');
 require('dotenv').config();
 
+const corsOptions = {
+  origin: 'http://localhost:3000',
+  credentials: true,
+  optionsSuccessStatus: 200, // some legacy browsers (IE11, various SmartTVs) choke on 204
+};
+
 const app = express();
 app.use(express.json());
-app.use(cors());
+
+app.use(cors(corsOptions));
+app.use(passport.initialize());
+app.use(passport.session());
```

Comment:
In order for passport sessions to work, you will need to also `yarn add express-session` and look into `app.use(session({...}))`. 
Have you guys tested your authentication with other requests yet? If not, I recommend setting up a dummy "isAuthenticated" endpoint to test and make sure that all your authentication is working and persists across multiple requests to the backend